engine/test/integration_tests/dm_tls/run.sh

```diff
@@ -46,8 +66,13 @@ function run() {
 	sed -i "s,<mysql2-key>,$(base64 -w0 $WORK_DIR/mysql2/client-key.pem)," $WORK_DIR/job.yaml
 	sed -i "s,<mysql2-cert>,$(base64 -w0 $WORK_DIR/mysql2/client-cert.pem)," $WORK_DIR/job.yaml
 
+	# wait executor online
+	sleep 20
```

Comment:
Can use OpenAPI to verify all executors are online 
https://github.com/pingcap/tiflow/blob/6587d4ebb46baeeecf14158a14385f6f7870ef74/engine/pkg/openapi/apiv1.swagger.json#L40

Reply:
https://github.com/pingcap/tiflow/pull/7091/commits/f4c5ed92c4cc6838493c36f51624b232d29074fc